app/templates/views/features.html

```diff
@@ -10,69 +10,66 @@
-<p class="govuk-body">Notify lets you:</p>
+<h2 class="heading-medium">Unlimited free emails</h2>
 <ul class="govuk-list govuk-list--bullet">
-  <li>set different permission levels for each team member</li>
-  <li>invite team members who do not have a government email address</li>
-  <li>choose who else can manage team members</li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_send_files_by_email') }}">Send files by email</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_email_branding') }}">Add email branding</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_reply_to_email_address') }}">Choose a reply-to email address</a></li>
 </ul>
 
-<h2 class="heading-medium" id="performance">Performance</h2>
-<p class="govuk-body">Notify commits to:</p>
+<h2 class="heading-medium">Send and receive text messages</h2>
 <ul class="govuk-list govuk-list--bullet">
-  <li>sending 95% of emails and text messages within 10 seconds</li>
-  <li>printing and posting letters by 3pm the next working day (if you send them to us before 5:30pm)</li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing', _anchor='text-messages') }}">Send thousands of free text messages</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_receive_text_messages') }}">Receive text messages</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing'), _anchor='international-numbers') }}">Send text messages to international numbers</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_text_message_sender') }}">Tell recipients who your messages are from</a></li>
 </ul>
 
-<p class="govuk-body">We send messages through several different providers. If one provider fails, Notify switches to another so that your messages are not affected.</p>
-
-<p class="govuk-body">Visit our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.performance') }}">performance data</a> page to see how Notify is performing.</p>
+<h2 class="heading-medium">Print and post letters</h2>
+<ul class="govuk-list govuk-list--bullet">
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing'), _anchor='letters') }}">Choose your postage</a></li>
+  <li>Send international letters</li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_letter_branding') }}">Add letter branding</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_upload_a_letter') }}">Upload a letter from a PDF</a></li>
+</ul>
 
-<h2 class="heading-medium" id="security">Security</h2>
-<p class="govuk-body">Notify protects and manages data to meet the needs of government services.</p>
+<h2 class="heading-medium">Automated sending</h2>
```

Comment:
I wonder if ‘automatic’ reads more simply than ‘automated’

```suggestion
<h2 class="heading-medium">Automatic sending</h2>
```

Reply:
I like automatic - to me it emphasises a user not having to actually do anything more than automated does and is probably more recognisable than automated.